packages/notifi-react-example/package.json

```diff
@@ -47,10 +47,10 @@
   },
   "scripts": {
     "start": "BROWSER=none react-app-rewired start",
-    "test": "cypress run --component --spec 'cypress/component/NotifiSubscriptionCard.cy.tsx' --browser chrome",
+    "test": "cypress run --component --spec 'cypress/component/NotifiSubscriptionCard.cy.tsx' --browser firefox",
```

Comment:
was this change expected?

Reply:
Yeah, I had changed that because Chrome is not installed on my system. I just put in a commit to change it back.